test_expat_deb_2_2_0 is failing

In #125 I set up some new file tests and test_expat_deb_2_2_0 is not working.

The expat checker works fine on the expat rpm test, but fails on the deb one.  My guess is that there's some difference in the version string so we'll need to add another option.

```
======================================================================
FAIL: test_expat_deb_2_2_0 (test.test_scanner.TestScanner)
Test detection of expat 2.2 debian package
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/terri/Code/cve-bin-tool/test/test_scanner.py", line 274, in test_expat_deb_2_2_0
    '2.2.0')
  File "/home/terri/Code/cve-bin-tool/test/test_scanner.py", line 109, in _file_test
    self.assertIn(package, cves)
AssertionError: 'expat' not found in {}

----------------------------------------------------------------------
```

--- a/test/test_scanner.py
+++ b/test/test_scanner.py
@@ -213,7 +213,7 @@ class TestScanner(unittest.TestCase):
         """ Test detection of expat 2.2 debian package """
         self._file_test(
             "http://http.us.debian.org/debian/pool/main/e/expat/",
-            "expat_2.2.0-2+deb9u1_amd64.deb",
+            "libexpat1_2.2.0-2+deb9u1_amd64.deb",
             "expat",
             "2.2.0",
         )

